(docs) - Apply edits to automatically flagged errors (#1437)
FormidableLabs/urql: docs/basics/errors.md

- The `networkError` property will contain any error that stopped `urql` from making a network
  request.
- The `graphQLErrors` property may be an array that contains [normalized `GraphQLError`s as they

>   were received in the `errors` array from a GraphQL API.](https://graphql.org/graphql-js/error/)

Reviewer: ```suggestion
  are returnerd in the `errors` array from a GraphQL API.](https://graphql.org/graphql-js/error/)
```
